x/gateway/keeper/query_params_test.go

```diff
@@ -0,0 +1,20 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	testkeeper "github.com/pokt-network/poktroll/testutil/keeper"
+	"github.com/pokt-network/poktroll/x/gateway/types"
+)
+
+func TestParamsQuery(t *testing.T) {
+	keeper, ctx := testkeeper.GatewayKeeper(t)
+	params := types.DefaultParams()
```

Comment:
Similar question to https://github.com/pokt-network/poktroll/pull/371#discussion_r1493857767; does it make sense to assert that params are initially empty?